chopsticks-dstream/src/main/scala/dev/chopsticks/dstream/Dstreams.scala

```diff
@@ -54,16 +54,19 @@ object Dstreams extends LoggingContext {
       }
     } yield binding
 
-    ZManaged.make(
-      acquire
-        .logResult("dstream server startup", b => s"Dstream server bound: ${b.localAddress}")
-    ) { binding =>
-      Task
-        .fromFuture(_ => binding.terminate(10.seconds))
-        .log("dstream server teardown")
-        .map(_ => ())
-        .catchAll(e => ZLogger.error("Failed unbinding dstream server", e))
-    }
+    ZManaged
+      .make(
+        acquire
+          .logResult("dstream server startup", b => s"Dstream server bound: ${b.localAddress}")
+      ) { binding =>
+        ZIO.access[IzLogging](_.get).map(_.zioLogger).flatMap { zLogger =>
```

Comment:
`ZIO.access[IzLogging](_.get.zioLogger)` should work

Reply:
Done